easybuild/easyconfigs/c/cairomm/cairomm-1.14.4-GCC-11.3.0-cairo-1.17.4.eb

```diff
@@ -0,0 +1,39 @@
+# Updated to MesonNinja as the autogen.sh complained.
+# Author: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'MesonNinja'
+
+name = 'cairomm'
+version = '1.14.4'
```

Comment:
and do we really need a slightly older version as well? Introducing multiple deps are always just a huge pain in the end, and unless there is a very strong reason, i would say to just avoid it.

Reply:
YES! There is a change of the API between 1.14.x and 1.16.x which stopped me from building `graph-tool` (PR #17389) 
See also [here](https://forum.skewed.de/t/compilation-error-with-cairomm-1-16/1397)
So I am afraid, we need to stick with that or we kick out the 1.16 version from the toolchain, **not** the 1.14 one.

Reply:
1.14.4 removed, so no longer relevant